<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectsCardDialog</class>
 <widget class="QDialog" name="ProjectsCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проект</string>
  </property>
  <widget class="QDialogButtonBox" name="m_buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_nameText">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>551</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_descriptionText">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_chiefLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>301</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Руководитель:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_chiefsList">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>353</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Плановая дата завершения:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="m_planDateEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Фактическая дата завершения:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="m_factDateEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectsCardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectsCardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
